src/data/faq_de.json

```diff
@@ -158,11 +158,23 @@
                             "Zeigt die App nach Prüfung an, dass keine Einreiseregeln vorhanden sind, können Sie sich unter <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/de</a> über die Regeln informieren."
                         ]
                     },
+                    {
+                        "title": "Warum zeigt die Corona-Warn-App auf Version 2.8 an, dass mein Zertifikat nicht prüfbar ist, obwohl die vorherige Version mein Zertifikat als gültig anzeigte?",
+                        "anchor": "dcc_validity_check_change",
+                        "active": true,
+                        "textblock": [
+                            "Mit der Version 2.8 hat sich das Verhalten der Zertifikatsgültigkeitsprüfung geändert. Einreiseregeln für Reisende wurden von allen EU-Mitgliedsländern aufgestellt, aber bis heute haben nur wenige Länder technische Einreiseregeln bereitgestellt, die die Corona-Warn-App nutzen könnte. Vor Version 2.8, wenn für ein bestimmtes Land keine Einreisebestimmungen zur Verfügung standen, zeigte die Corona-Warn-App nach einer Überprüfung 'Gültiges Zertifikat' an und erklärte im folgenden Text, dass für dieses Land keine Einreisebestimmungen zur Verfügung gestellt wurden/überprüft werden konnten. 'Gültiges Zertifikat' bezog sich in diesem Fall auf die ’Technische Gültigkeit' (mehr Informationen darüber, was die ‚Technische Gültigkeit' ist, finden Sie in <a href='#eu_dcc_validity' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ-Eintrag).",
+                            "Wir haben jedoch festgestellt, dass dieses Ergebnis fehlinterpretiert werden kann, z.B. wenn eine unvollständige Impfung überprüft wurde, was zu der falschen Schlussfolgerung führen könnte, dass dieses Zertifikat eine uneingeschränkte Einreise rechtfertigt. Dies haben wir mit der Version 2.8 klargestellt: Für den Fall, dass ein Mitgliedsland keine technische Einreisebestimmungen für die Corona-Warn-App zur Verfügung gestellt hat, lautet die Überschrift nun ‚Zertifikat nicht prüfbar', während die folgenden Texte gleich geblieben sind. Dies bedeutet <b>nicht</b>, dass Ihr Zertifikat ungültig ist oder dass Sie nicht in das Land einreisen dürfen, für das Sie Ihr Zertifikat überprüft haben. In diesem Fall überprüfen Sie bitte Ihren Status anhand der Informationen, die Sie unter <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/de</a> finden. Weitere Informationen zu dieser Änderung finden Sie im zweiten Absatz dieses Blog Posts: <a href='../blog/2021-08-25-cwa-version-2-8/' target='_blank' rel='noopener noreferrer'>Mit Version 2.8 der Corona-Warn-App werden Namen in Zertifikaten in standardisierter Schreibweise angezeigt</a>.<hr>",

```

Comment:
technische Einreiseregeln => Einreiseregeln in einem maschinenlesbaren Format
Einreisebestimmungen => Einreiseregeln (a couple of times - to align with all previous usage)

Vor Version 2.8, wenn für ein bestimmtes Land keine Einreise`regeln` `in einem maschinenlesbaren Format` zur Verfügung standen, zeigte die Corona-Warn-App nach einer Überprüfung 'Gültiges Zertifikat' an und erklärte im folgenden Text, dass für dieses Land keine Einreise`regeln` zur Verfügung gestellt wurden/überprüft werden konnten. ...

könnte => konnte & some additions/rephrasing
"Wir haben jedoch festgestellt, dass dieses Ergebnis fehlinterpretiert werden kann, z.B. wenn eine unvollständige Impfung überprüft wurde, was zu der falschen Schlussfolgerung führen `konnte`, dass `für `dieses Zertifikat `keine` ~uneingeschränkte~ Einreise`beschränkungen bestehen` ~rechtfertigt~. Dies haben wir mit der Version 2.8 klargestellt: Für den Fall, dass ein Mitgliedsland keine ~technische~ Einreise`regeln in einem maschinenlesbaren Format` für die Corona-Warn-App zur Verfügung gestellt hat, ...